mmdeploy/codebase/mmpose/deploy/pose_detection_model.py

```diff
@@ -145,11 +147,11 @@ def pack_result(self,
             # convert keypoint coordinates from input space to image space
             if convert_coordinate:
                 input_size = data_sample.metainfo['input_size']
-                bbox_centers = gt_instances.bbox_centers
-                bbox_scales = gt_instances.bbox_scales
+                input_center = data_sample.metainfo['input_center']
```

Comment:
other models like `hrnet` fails in here:
```
Traceback (most recent call last):
  File "/root/workspace/mmdeploy/mmdeploy/utils/utils.py", line 41, in target_wrapper
    result = target(*args, **kwargs)
  File "/root/workspace/mmdeploy/mmdeploy/apis/visualize.py", line 72, in visualize_model
    result = model.test_step(model_inputs)[0]
  File "/usr/local/lib/python3.8/dist-packages/mmengine/model/base_model/base_model.py", line 145, in test_step
    return self._run_forward(data, mode='predict')  # type: ignore
  File "/usr/local/lib/python3.8/dist-packages/mmengine/model/base_model/base_model.py", line 340, in _run_forward
    results = self(**data, mode=mode)
  File "/usr/local/lib/python3.8/dist-packages/torch/nn/modules/module.py", line 1501, in _call_impl
    return forward_call(*args, **kwargs)
  File "/root/workspace/mmdeploy/mmdeploy/codebase/mmpose/deploy/pose_detection_model.py", line 116, in forward
    results = self.pack_result(preds, data_samples)
  File "/root/workspace/mmdeploy/mmdeploy/codebase/mmpose/deploy/pose_detection_model.py", line 150, in pack_result
    input_center = data_sample.metainfo['input_center']
KeyError: 'input_center'

```

Reply:
The post-processing modification for top-down models is now compatible with the `dev-1.x` branch of mmpose. This branch will be merged into the `main` branch soon